cargo-shuttle/src/lib.rs

```diff
@@ -796,6 +802,10 @@ impl Shuttle {
                         trace!(status = ?err, "stopping the runtime errored out");
                     });
             }
+            println!(
+                "Run `cargo shuttle project start` to create a project environment on Shuttle.\n\
+                 Run `cargo shuttle deploy` to deploy your Shuttle service."
+            );
```

Comment:
I would add this `println!` right above the `if signal_received`. We want to display it when doing a `local_run`, but not only if a signal was sent.